src/components/tradingviewChart/utils.ts

```diff
@@ -252,4 +252,9 @@ export const pairsToExchangeMapping: { [key: string]: string } = {
   BNTWETH: 'COINBASE:BNTETH',
   COTETH: 'UNISWAP3ETH:',
   COTWETH: 'UNISWAP3ETH:',
+  SEIUSDC: 'BINANCE:SEIUSDC',
```

Comment:
might be worth adding this to the config as well, I like that popular pairs are in the config now

Reply:
Moved the pairsToExchangeMapping to config/utils